sql-select/second-store-customers.sql

```diff
@@ -0,0 +1,7 @@
+SELECT "customerId",
+       "firstName",
+       "lastName",
+       "email",
+       "storeId"
+  FROM "customers"
+ WHERE "storeId" = 2
```

Comment:
You have too many fields being selected:

---

![image](https://user-images.githubusercontent.com/16143771/179878815-127b8844-6fc7-4806-8bec-ec6d3f6ff1be.png)